Content/Data/Audio-External.xml

```diff
@@ -1,13 +1,120 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-
-<data>
+<data content="Audio\Emotes\Male\Burp">
```

Comment:
this seems to have been disconnected from the burp on like 6

Reply:
content="Audio\Emotes\Male\Burp" I think this part gets moved to line 6

Reply:
Line 2 pointed to a file not a folder, and Male Burp content target was missing.   Thats not how I left it.

There are strange things afoot at the GitHub.